migrations/2_MessageStateTransitionTable.js

```diff
@@ -0,0 +1,34 @@
+module.exports = {
+  up: function(query, Sequelize) {
+    return query.createTable('MessageStateTransition', {
+      id: {
+        primaryKey: true,
+        type: Sequelize.UUID,
+        defaultValue: Sequelize.UUIDV4,
```

Comment:
What is the difference between using UUID and UUIDV4?

Reply:
UUID is the type of the column, and UUIDV4 is generator of UUIDs (which is why it's the default value, since it's used to create the UUID for each message). The same process is used with the other columns that use UUIDs.